docs/source/reference/numpysupported.rst

```diff
@@ -210,7 +210,7 @@ The following methods of Numpy arrays are supported:
 * :meth:`~numpy.ndarray.repeat` (no axis argument)
 * :meth:`~numpy.ndarray.reshape` (only the 1-argument form)
 * :meth:`~numpy.ndarray.sort` (without arguments)
-* :meth:`~numpy.ndarray.sum` (with or without the ``axis`` argument)
+* :meth:`~numpy.ndarray.sum` (with or without the ``axis`` argument. ``axis`` only supports scalar values)
```

Comment:
I think s/scalar/``integer`` to be really explicit. Also, this line needs wrapping.